app/bower.json

```diff
@@ -19,7 +19,7 @@
         "oclazyload": "^1.0.9",
         "ngSafeApply": "./libs/Scope.SafeApply.js",
         "izitoast": "^1.1.3",
-        "contenteditor": "https://sunbirdstagingpublic.blob.core.windows.net/sunbird-content-staging-knowlg/content-editor/scripts/base-editor.min.js"
+        "contenteditor": "https://objectstorage.ap-mumbai-1.oraclecloud.com/n/bmzbbujw9kal/b/odev-dev-diksha-contents/o/content-editor/scripts/base-editor.min.js"
```

Comment:
@ddevadat Please revert this change. This OCI specific change not required in sunbird.

Reply:
will look into this

Reply:
@pallakartheekreddy its removed. please review

Reply:
This PR is changes not required in Sunbird